### ISSUE TITLE
Does the loss function's value range typically extend from 0 to positive infinity?

### ISSUE THREAD

**Issue by InTheFuture7** [ioannislivieris/Grad_CAM_Siamese#2]
```python
# setup loop with TQDM and dataloader
loop = tqdm(test_dl, leave=True)
# setup epoch's metrics
metrics = {'losses': [], 'accuracy': [], 'AUC': []}
for step, (img1, img2, labels) in enumerate(loop):
    img1, img2, labels = img1.to(device), img2.to(device), labels.to(device)
    # Get loss & predictions
    predictions, loss = model(img1, img2, labels)  # here
    # Calculate performance metrics
    accuracy, AUC, _ = performance_evaluation(labels, predictions)               
    # Add loss/accuracy/AUC
    metrics['losses'].append(loss.item())
    metrics['accuracy'].append(accuracy)
    metrics['AUC'].append(AUC)   

```

**Comment by InTheFuture7** [comment]
Perhaps the range of values for the loss function may be [0, +∞].

![Snipaste_2024-05-31_21-27-35](https://github.com/ioannislivieris/Grad_CAM_Siamese/assets/65111660/7f94fa41-d230-4dee-b253-c780ed7c24bf)
